### PR TITLE
Add court codes

### PR DESCRIPTION
**Pull request by cc271793784**

### CODE REVIEW

File: docassemble/MACourts/data/sources/housing_courts_addcode+add4new.json [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+      "address": "Barnstable, MA"
+    },
+    "description": "The barnstable division of the Housing court is located in Barnstable, MA"
+  },
+  {
+    "fax": "",
+    "code": "H84",
+    "name": "Eastern Housing Court - Chelsea Session",
+    "phone": "(617)788-8485",
+    "has_po_box": false,
```
Comment (nonprofittechy) [inline docassemble/MACourts/data/sources/housing_courts_addcode+add4new.json:513]: can we add address (from Chelsea District Court) and description from this page: https://www.mass.gov/locations/eastern-housing-court-chelsea-session

**Comment by nonprofittechy** [comment]
@cc271793784 this is great. Just one minor change: can you make sure the new court locations include addresses and descriptions? See my inline comment for eastern housing court - Chelsea Session. Nice work!

**Comment by cc271793784** [comment]
> @cc271793784 this is great. Just one minor change: can you make sure the new court locations include addresses and descriptions? See my inline comment for the eastern housing court - Chelsea Session. Nice work!

Hi Quinten. I just quickly check it. There's something missing in the file I download from the JSON editor (I do add description and address info for Chelsea session). I'll re-download it and make sure everything is correct

**Comment by plocket** [comment]
Fixes #14 and fixes #15?

**Comment by cc271793784** [comment]
#14 and #16 has been fixed and the name changed from "code" to "court_code"

File: docassemble/MACourts/macourts.py [hunk 2]
```diff
@@ -421,12 +421,14 @@ def load_courts_from_file(self, court_name, data_path='docassemble.MACourts:data
 
         path = path_and_mimetype(os.path.join(data_path,json_path+'.json'))[0]
 
-        with open(path) as courts_json:
+        with open(path,encoding='utf-8-sig') as courts_json:
```
Comment (nonprofittechy) [inline docassemble/MACourts/macourts.py:424]: I think this change is safe. But we should avoid changing unrelated code when possible. Alternative is to remove BOM from the JSON files as that does not seem to be part of the spec.

This is the only clue I found about safety, but I'm satisfied: https://github.com/psychopy/psychopy/pull/2168
